src/Orchard.Cms.Web/Startup.cs

```diff
@@ -40,9 +38,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseStaticFiles();
             loggerFactory.AddConsole(Configuration);
```

Comment:
It is only to add the new target in \src\Orchard.Cms.Web\NLog.config file. We don't need an extra module, do we?